scala3doc/src/dotty/renderers/ScalaSearchbarDataInstaller.scala

```diff
@@ -0,0 +1,58 @@
+package dotty.dokka
+
+import com.fasterxml.jackson.module.kotlin.ExtensionsKt._
+import org.jetbrains.dokka.base.renderers.html.{SearchbarDataInstaller, SearchRecord}
+import java.util.{List => JList}
+import java.util.concurrent.ConcurrentHashMap
+import collection.JavaConverters._
+import org.jetbrains.dokka.pages._
+import dotty.dokka.model.api._
+import org.jetbrains.dokka.plugability._
+import org.jetbrains.dokka.plugability.DokkaContext
+import org.jetbrains.dokka.plugability.DokkaPluginKt._
+import org.jetbrains.dokka.base.DokkaBase
+import dotty.dokka.PluginUtils._
+import org.jetbrains.dokka.base.signatures.SignatureProvider
+import org.jetbrains.dokka._
+import org.jetbrains.dokka.model._
+import scala.collection.concurrent.TrieMap
+
+class ScalaSearchbarDataInstaller(val ctx: DokkaContext) extends SearchbarDataInstaller:
+
+  case class PageEntry(val name: String, val signature: String, val link: String, val pkg: String)
+
+  val pages = TrieMap[String, PageEntry]()
+
+  val signatureProvider = querySingle[DokkaBase, SignatureProvider](ctx, _.getSignatureProvider)
+
+  override def processPage(page: ContentPage, link: String) =
+    Option(page.getDocumentable) match {
+      case Some(member) => processMember(member, link)
+      case None => ()
```

Comment:
We need to add support for titles from static documentation nodes

Reply:
added support